at_dome/setup.sh

```diff
@@ -1,30 +1,6 @@
-#!/usr/bin/env bash
-
-# Source this file when starting the container to set it up
-
-echo "#"
-echo "# Loading LSST Stack"
-. /opt/lsst/software/stack/loadLSST.bash
-setup lsst_distrib
-echo "#"
-echo "# Loading sal environment"
-. repos/ts_sal/setup.env
-echo "#"
-echo "# Setting up sal, salobj and scriptqueue"
-
-setup ts_xml -t current
-setup ts_sal -t current
-setup ts_salobj -t current
-setup ts_scriptqueue -t current
-setup ts_config_attcs -t current
-setup ts_ATDome -t current
-
-cd /home/saluser/repos/ts_ATDome/bin/
-
-while :
-    do
-        echo "# Starting ATDome CSC"
-        python ./run_atdome.py
-done
 
+source ~/miniconda3/bin/activate
+source $OSPL_HOME/release.com
+source /home/saluser/.bashrc
 
+run_atdome.p
```

Comment:
This looks like the extension is messed up.

Reply:
I am actually overriding this at launch time but you are right! better not store broken things. I corrected it.